src/bin/common.rs

```diff
@@ -368,12 +371,12 @@ fn load_speed_test_cfg(setting: &str) -> EncoderConfig {
     "min_block_size_8x8" => {
       cfg.speed_settings.min_block_size = BlockSize::BLOCK_8X8;
     },
-    "min_block_size_16x16" => {
```

Comment:
Breaking script compatibility isn't great :) in this case is probably not important, but keep it in mind in the future.

Reply:
Yeah, in this case it's only because I'm reasonably sure nothing is using this yet.

Reply:
More specifically, these changes came out of trying to create a script that uses this feature, and finding the limitations.